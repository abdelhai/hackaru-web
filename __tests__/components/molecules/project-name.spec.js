import Factory from '@/__tests__/__setups__/factory';
import ProjectName from '@/components/molecules/project-name';

describe('ProjectName', () => {
  it('render correctly', () => {
    const wrapper = new Factory(ProjectName, {
      propsData: {
        name: 'Development',
        color: '#ff0'
      }
    }).shallow();
    expect(wrapper.element).toMatchSnapshot();
  });
});
