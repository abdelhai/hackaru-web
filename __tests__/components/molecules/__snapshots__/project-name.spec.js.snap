// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProjectName render correctly 1`] = `
<div
  class="project-name"
>
  <dot-stub
    class="dot"
    color="#ff0"
  />
  
  Development

</div>
`;
